lemarche/www/pages/urls.py

```diff
@@ -160,6 +161,11 @@
     path("track/", TrackView.as_view(), name="track_frontend"),
     # Calculator endpoints
     path("calibrer-achat-socialement-responsable/", ImpactCalculatorView.as_view(), name="impact_calculator"),
+    path(
+        "calculez-limpact-social-dun-achat-inclusif/",
+        SocialImpactBuyersCalculatorView.as_view(),
+        name="social_impact-buyers_calculator",
```

Comment:
```suggestion
        name="buyer_social_impact_calculator",
```
pour matcher avec `impact_calculator` plus haut ?